gallery.html

```diff
@@ -4,8 +4,24 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Gallery</title>
+    <link rel="stylesheet" href="index.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Titillium+Web&display=swap" rel="stylesheet">
 </head>
 <body>
-    
+    <section class="gallery">
+        <h1>Gallery</h1>
+        <div class="pictures">
+            <img src="images/deadmau5.jpg" alt="concert">
```

Comment:
I assume the same image is being used repeatedly, knowing the primary intent is showing the ability to display and position images successfully.  However, even though that's the intent, I would expect the 'alt' attributes to be unique to demonstrate the type of information you would provide.